docs/docs/recipes.md

```diff
@@ -189,3 +189,69 @@ since `0.7.4`).  Requires Mesos version `0.20.0` or later.
   "uris": ["http://downloads.mesosphere.com/misc/toggle.tgz"]
 }
 ```
+
+## Constraining task placement / static partitioning
+
+Usually, Mesos and Marathon collaboratively decide where to start your application tasks without your interference.
+That is desirable because it truly makes your infrastructure easily scalable. All resources are treated equally. 
+However, there are times when you have
+to separate a few machines from the common pool and ascertain that only some special tasks are scheduled there. For
```

Comment:
I'd leave out _a few_ – I can imagine assigning half of the cluster to a specific use.